lib/modulesync/renderer.rb

```diff
@@ -30,12 +30,13 @@ def self.render(_template, configs = {})
       ForgeModuleFile.new(configs).render
     end
 
-    def self.sync(template, target_name)
+    def self.sync(template, target_name, mode)
```

Comment:
I'd prefer `mode=nil` and adding an `if` guard to `File.chmod`

Reply:
I just pushed an attempt to do this.